comps/cores/mega/gateway.py

```diff
@@ -837,6 +838,9 @@ def parser_input(data, TypeClass, key):
 
 
 class MultimodalQnAGateway(Gateway):
+    asr_port = int(os.getenv("ASR_SERVICE_PORT", 3001))
+    asr_endpoint = os.getenv("ASR_SERVICE_ENDPOINT", "http://0.0.0.0:{}/v1/audio/transcriptions".format(asr_port))
```

Comment:
Is `ASR_SERVICE_ENDPOINT` documented anywhere? I know the `ASR_SERVICE_PORT` is not new.
Or maybe you have it documented in the corresponding GenAIExamples PR.

Reply:
It is in the corresponding GenAIExamples PR, e.g. here: https://github.com/opea-project/GenAIExamples/pull/1225/files#diff-1d1cfced259dada6f277c25da44cbaea85aa3629b3ce3fb54780c4525a9a3896R83, but if there's another place in this repo where it should be added please let me know.